package.json

```diff
@@ -21,6 +21,7 @@
     "eslint": "^6.6.0",
     "eslint-plugin-prettier": "^3.1.4",
     "eslint-plugin-react": "^7.20.6",
+    "gsap": "^3.6.1",
     "prettier": "^2.1.1",
```

Comment:
gsap library added to package.json to handle animations

Reply:
Thank you for the explanation on what this was for.